packages/web-app-files/src/composables/extensions/useFileSideBars.ts

```diff
@@ -227,7 +228,10 @@ export const useSideBarPanels = () => {
               }
               if (
                 isLocationTrashActive(router, 'files-trash-generic') ||
-                isLocationPublicActive(router, 'files-public-link')
+                isLocationPublicActive(router, 'files-public-link') ||
+                isLocationSharesActive(router, 'files-shares-with-others') ||
```

Comment:
Since we don't support resharing anymore we can be certain, that files listed on `shared with others` or `shared via link` are our own files, so those two routes should actually show the versions panel.